Examples/VaporDemo/Sources/App/Views.swift

```diff
@@ -10,14 +11,21 @@ struct MainPage: HTMLDocument {
         meta(.charset(.utf8))
         script(.src("/htmx.min.js")) {}
         script(.src("/htmxsse.min.js")) {}
+        script(.src("/htmxws.min.js")) {}
     }
 
     var body: some HTML {
         header {
             h2 { "Vapor + Elementary + HTMX Demo" }
+            // example of using htmx sse
             div(.hx.ext(.sse), .sse.connect("/time")) {
                 p(.sse.swap("time")) { "Server Time:" }
             }
+            // example of using htmx ws
+            div(.hx.ext(.ws), .ws.connect("/echo"), .hx.vals("{ \"value\": \"TestValue\"}"), .hx.swapOOB(.innerHTML), .hx.target("#echo"), .class("flex justify-between")) {
```

Comment:
for json strings `#"..."#` would be nicer
or ideally have the data come from a text input or something